library-model/library-model-generator/src/main/java/com/uber/nullaway/libmodel/LibraryModelGenerator.java

```diff
@@ -157,7 +155,7 @@ public Result process(Path localPath, Path absolutePath, ParseResult<Compilation
 
   private static class AnnotationCollectionVisitor extends VoidVisitorAdapter<Void> {
 
-    private String packageName = "";
+    private StringBuilder parentName;
```

Comment:
Probably better to just make this a `String`?  I think making it a `StringBuilder` is an unnecessary optimization and makes the code more brittle